### DISCUSSION TITLE
CC2652P7 support

### DISCUSSION THREAD

**Discussion by corogoo** [Koenkk/zigbee2mqtt#15698] [Devices]
RF-STAR has launched the RF-BM-2652P4 module, which uses the CC2652P7 chip. Is there any plan to support this module in the future?

**Comment by Koenkk** [comment]
Until USB dongles become widely available not, you can always compile your own firmware based on https://github.com/koenkk/z-stack-firmware

**Reply by catduckgnaf** [reply]
I can confirm this chipset works.

**Reply by Bosemani** [reply]
Hai @catduckgnaf ,
   I'm also trying using CC2652P7 chip. I'm having some doubts.

1. Did you compiled bin file by following this steps? [link](https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/COMPILE.md)
2. Did you used RF-BM-2652P4  Module or your own Design using CC2652P7 chip?

TIA

**Reply by catduckgnaf** [reply]
I bought from a trusted maker who put it together for me with POE. It was a test run. The May 2023 firmware supports it.

**Reply by Bosemani** [reply]
Hai @catduckgnaf 
Can you update ASAP

**Comment by Bosemani** [comment]
Hai @catduckgnaf 

Thank you for response, I have few questions.
Which firmware did you used? CC1352P2_CC2652P_launchpad_coordinator_ ? or any other?
What is maximum TX power you used?
